Bottom: 87d8e1e280225dc51509235a1991d5d5dbc1ada9
Top:    cd9261e8539de696ade81e58210f2e862c55d25c
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-24 14:27:40 +0800

Refresh of autofs-5.0.7-use-LIBS-for-link-libraries.patch

---

diff --git a/daemon/Makefile b/daemon/Makefile
index 9e9d635..6604c9a 100644
--- a/daemon/Makefile
+++ b/daemon/Makefile
@@ -20,7 +20,7 @@ CFLAGS += -DAUTOFS_FIFO_DIR=\"$(autofsfifodir)\"
 CFLAGS += -DAUTOFS_FLAG_DIR=\"$(autofsflagdir)\"
 CFLAGS += -DVERSION_STRING=\"$(version)\"
 LDFLAGS += -rdynamic
-LIBS = -ldl
+LIBS += -ldl
 
 ifeq ($(LDAP), 1)
     CFLAGS += $(XML_FLAGS)
